pkg/agent/agent.go

```diff
@@ -795,11 +794,10 @@ func (i *Initializer) initNodeLocalConfig() error {
 	}
 	transportIPv4Addr = nodeIPv4Addr
 	transportIPv6Addr = nodeIPv6Addr
-	transportInterfaceName = localIntf.Name
+	transportInterface := localIntf
```

Comment:
why do we have both `transportInterface` and `localIntf` variables? Should we just have `transportInterface`?

Reply:
I wanted to avoid assign return values got from `getIPNetDeviceFromIP(ipAddrs)` to nodeIPv4Addr, nodeIPv6Addr and transportInterface in one line because the 1st and 2nd variables are related to the Node IP interface but the 3rd variable is transport interface. and the above 3 lines can express that transport IPs and transport interface default to node IPs and node interface.

I have adjusted the code a little and added some comments to make it more clear, PTAL.